### ISSUE TITLE
Can't resolve 'react-highlighter'

### ISSUE THREAD

**Issue by hyzhak** [ericgio/react-bootstrap-typeahead#261]
### Version
`2.0.0-rc.3`

### Steps to reproduce

```
//install the last version of lib
npm install react-bootstrap-typeahead@2.0.0-rc.3
//import lib
import {Typeahead} from 'react-bootstrap-typeahead';
//use it somewhere as usual
<Typeahead/>
//and compile
```

### Expected Behavior

```
compile without errors
```

### Actual Behavior

```
ERROR in ./node_modules/react-bootstrap-typeahead/lib/TypeaheadMenu.react.js
Module not found: Error: Can't resolve 'react-highlighter' in '/node_modules/react-bootstrap-typeahead/lib'
 @ ./node_modules/react-bootstrap-typeahead/lib/TypeaheadMenu.react.js 15:24-52
 @ ./node_modules/react-bootstrap-typeahead/lib/Typeahead.react.js
 @ ./node_modules/react-bootstrap-typeahead/lib/index.js
...
```

**Comment by ericgio** [comment]
What the fuck. NPM keeps bundling stale files...

react-highlighter was removed as a dependency in v2.0.0-rc.3 but for some reason it looks like an older version of `TypeaheadMenu` was included [in the package](https://registry.npmjs.org/react-bootstrap-typeahead/-/react-bootstrap-typeahead-2.0.0-rc.3.tgz). I have no idea why this is happening. The issue was also referenced in #258.

**Comment by ericgio** [comment]
Just published v2.0.0-rc.4 and confirmed that the new files are in there. Thanks for the report and sorry about the mess, will try to get to the bottom of it.

**Comment by hyzhak** [comment]
no problem, man! Thanks for your work 👍
